wp-pgp-encrypted-emails.php

```diff
@@ -822,15 +1148,24 @@ public static function commentText ($text) {
      * @uses update_user_meta()
      *
      * @return void
-     */
-    public static function saveProfile ($user_id) {
+     */ function saveProfile ($user_id) {
```

Comment:
This line has (accidentally?) closed a comment and opened a function signature on the same line. It feels messy. :( Would you separate these? Thanks!